src/constants/index.ts

```diff
@@ -16,6 +16,14 @@ export const USDT = new Token(ChainId.MAINNET, '0xdAC17F958D2ee523a2206206994597
 export const COMP = new Token(ChainId.MAINNET, '0xc00e94Cb662C3520282E6f5717214004A7f26888', 18, 'COMP', 'Compound')
 export const MKR = new Token(ChainId.MAINNET, '0x9f8F72aA9304c8B593d555F12eF6589cC3A579A2', 18, 'MKR', 'Maker')
 export const AMPL = new Token(ChainId.MAINNET, '0xD46bA6D942050d489DBd938a2C909A5d5039A161', 9, 'AMPL', 'Ampleforth')
+export const SUSD = new Token(
+  ChainId.MAINNET,
+  '0x57Ab1ec28D129707052df4dF418D58a2D46d5f51',
```

Comment:
is this the correct proxy address which has uniswap liquidity?

Reply:
that's correct, `0x57ab1ec28d129707052df4df418d58a2d46d5f51` is the proxy and the implementation target is currently `0xae38b81459d74a8c16eaa968c792207603d84480`

- https://etherscan.io/address/0x57ab1ec28d129707052df4df418d58a2d46d5f51#readContract
- https://etherscan.io/address/0xae38b81459d74a8c16eaa968c792207603d84480#readContract
- https://uniswap.info/token/0x57ab1ec28d129707052df4df418d58a2d46d5f51